### ISSUE TITLE
Activity Log: always use object type name in description

### ISSUE THREAD

**Issue by keoshi** [Automattic/wp-calypso#17743]
To complement the use of icons, we should refine events language to make sure we’re always including the object type names (themes, plugin, etc.).

Examples:

![image](https://user-images.githubusercontent.com/390760/30025873-6b3ca4da-9172-11e7-8f61-5de8a7f61869.png)

> WP Super Cache plugin was autoupdated to version 1.5.4

![image](https://user-images.githubusercontent.com/390760/30025941-b897d358-9172-11e7-81ae-a65af4d77618.png)

> Cosimo theme was updated to version 1.3.2

**Comment by sirreal** [comment]
Would these read as follows?

* `The WP Super Cache plugin was autoupdated to version 1.5.4`
* `Cosimo theme was updated to version 1.3.2`

**Comment by michelleweber** [comment]
The two bulleted lines above read well and are clear. Theme names should ideally be italicized. 

Related: We should probably do a sweep and change autoupdate to auto-update; "autoupdate" isn't a word.

**Comment by sirreal** [comment]
Thanks @michelleweber!

> Theme names should ideally be italicized.

Did you exclude _plugin names_ intentionally, or should we italicize them as well and if possible?

**Comment by sirreal** [comment]
To be handled in D7165-code

**Comment by keoshi** [comment]
Yes @sirreal, I added the revised copy below each screenshot. We should always try use this format:

`[object_name] [object_type] was [action] [to version (optional)]`

**Comment by keoshi** [comment]
@michelleweber pinging you again for suggestions to finalize the language pattern used here.

I'd like to go for an updated revision on the above: `[object_type] [object_name] was [action] [(optional) to version]`.

Some examples might be:

> Page _Behind the scenes_ was published

> Plugin _Advanced Custom Fields_ was auto-updated to version 1.5.4

> Theme _Intergalactic 2_ was updated to version 2.0.1

> User _Kirsten Dunst_ was deleted

But @sirreal has a valid concern which is that we're not using active voice here and it may sound muddy. To understand the context we'll always have the actor in the same block as these descriptions though. Here's an example:

![image](https://user-images.githubusercontent.com/390760/30059672-f4999c68-9237-11e7-8f3a-70a0a9d3a088.png)

What are your thoughts? Thanks for you help!

**Comment by sirreal** [comment]
> To understand the context we'll always have the actor in the same block as these descriptions

Note, the "actor" may be a user, like "enej b" above, or something else, at the moment "Jetpack" is the only alternative.

**Comment by michelleweber** [comment]
1. Normally, it's just theme names that are italicized (e.g., in support docs, blog posts, etc). In this case though, italicizing the name of the thing that was updated/deleted helps make the update easier to read, so I'd vote for italicizing across the board, as in your example above.

2. This is a case where passive voice might be appropriate. It puts the emphasis is on the thing being updated or deleted, not on the actor.

To that end, I have a question: What's more important: to communicate that plugin X was updated, or to communicate that Enej is the one who did it? I.e., is the point of the log mainly to show me what happened, or is it to show me who did what? That dictates what makes the most sense from a language point of view.

**Comment by keoshi** [comment]
I'd say both are equally important. Maybe they'll have distinct priorities at different times, so it's hard to say one or the other.

Thanks, @michelleweber !

**Comment by michelleweber** [comment]
Good to know. In that case, let's default to active voice. 

[human actor] [action] the [thing]. ("Enej updated the _Twenty Seventeen_ theme.")

**Comment by sirreal** [comment]
Migrated to 84-gh-action-to-activity-log

**Comment by keoshi** [comment]
Just to be clear, @michelleweber: if we use that, we'll be repeating the actor on the left and right sides. On the left with the avatar, username, and role; on the right assigning the action again to that user.

My concern is that by using `[human actor] [action] the [thing].` all events will focus on the user first, despite the UI highlighting that already in its own column. Something like `[object_type] [object_name] was [action].` would arguably provide a better quick read since all `object_types` will be aligned vertically.

**Comment by keoshi** [comment]
See these screens of the two versions and the consistency across all events.

![image](https://user-images.githubusercontent.com/390760/30157674-d04708b2-93ba-11e7-90f9-c53cf7ef4369.png)

![image](https://user-images.githubusercontent.com/390760/30157777-1148c4cc-93bb-11e7-99cb-992658ca7f6c.png)

**Comment by michelleweber** [comment]
Sorry, you're totally clear and I should have read more closely. Totally agree with removing actor from the text, but would also suggest staying active -- [action] the [thing]. Published X page. Activated Y plugin. Plugin Jetpack has an update available.

This is an activity log, so these don't need to be full sentences -- unless there's a reason that I'm missing -- and it's clearer/easier to read if we simplify the wording.

**Comment by keoshi** [comment]
I like that mix and it complements the UI quite nicely. Thanks again, @michelleweber!
